local/run_metadata.bash

```diff
@@ -20,5 +20,4 @@ set +x
 docker_ip=$(ip -4 addr show docker0 | grep -P inet | head -1 | awk '{print $2}' | cut -d/ -f1)
```

Comment:
Probably lets move metadata.go and gcs.go in a emulator subdir, don't want too many files in local/ root dir ? thoughts ?

Reply:
subdir sounds better. Done!